FindPythonLibsNew.cmake

```diff
@@ -0,0 +1,288 @@
+# - Find python libraries
```

Comment:
@HDembinski Haven't been following this, but does this use pybind11? I see the file copied from pybind11 and if it's possible to use it from there, that would be better.

Reply:
Hello, @henryiii. Yes, this file is from pybind11. However, since it should be modified, because of the reasons described above. In short, we need to ignore `libpython` if it is not found, but the version of pybind11 uses default value which cannot be found on `manylinux` image.

Reply:
I'm quite sure pybind11's CMake files work on manylinux! I'll try to look into exactly what you needed here soon. (remind me if I forget)

Reply:
Basically, the problem is in this:

```cmake
if(NOT PYTHON_LIBRARY)
    #set(PYTHON_LIBRARY python${PYTHON_LIBRARY_SUFFIX}) <-- this is a problem
    # To fix it is replaced with:
    message(WARNING "Python library not found (this may be ok)")
 endif()
```